chapter_introduction/index_vn.md

```diff
@@ -1046,7 +1046,12 @@ It is treated with a different set of algorithms
 than those used for regression (although many techniques will carry over).
 -->
 
-*dịch đoạn phía trên*
+Trong khi các mô hình hồi quy hiệu quả cho việc trả lời các câu hỏi *bao nhiêu?*, rất nhiều bài toán không phù hợp với nhóm mô hình này.
+Ví dụ, một ngân hàng muốn thêm chức năng quét ngân phiếu trong ứng dụng di động của họ.
+Việc này sẽ bao gồm việc khách hàng chụp một bức ảnh của ngân phiếu với camera của điện thoại và mô hình học máy sẽ cần phải tự động hiểu nội dung chữ trong bức ảnh.
+Nó cũng cần hiểu chữ viết tay thậm chí một cách ổn định. <!-- need more dicussion-->
```

Comment:
```suggestion
Hiểu được cả chữ viết tay sẽ giúp ứng dụng hoạt động còn ổn định hơn nữa.
```

Reply:
@lkhphuc em dịch hơi sai nghĩa gốc. Ý tác giả là nó (mô hình) cần hiểu chữ viết tay một cách robost. Việc này khác với "sẽ robost hơn nếu nó hiểu chữ viết tay".

Reply:
Mình lại thấy hiểu giống @lkhphuc nhỉ. Đoạn to-be kia là bổ nghĩa cho cả đoạn phía trước là "hiểu được chữ viết tay" chứ nhỉ?

Reply:
Mình thấy nghĩa không khác nhiều nên sẽ chấp nhận suggestion của @lkhphuc.